packages/survey-angular-ui/src/comment-other.component.html

```diff
@@ -1,4 +1,4 @@
-<textarea *ngIf="!question.isReadOnlyRenderDiv()" [id]="otherId" [attr.maxlength]="question.getOthersMaxLength()"  [attr.aria-required]="question.ariaRequired" [attr.aria-label]="question.ariaLabel" [attr.placeholder]="otherPlaceholder" 
+<textarea *ngIf="!question.isReadOnlyRenderDiv()" [id]="otherId" [attr.maxlength]="question.getOthersMaxLength()"  [attr.aria-required]="question.ariaRequired || question.a11y_input_ariaRequired" [attr.aria-label]="question.ariaLabel || question.a11y_input_ariaLabel" [attr.placeholder]="otherPlaceholder" 
```

Comment:
Why do we have two checks - question.ariaRequired || question.a11y_input_ariaRequired
The quesion is required or not required. Why a11y_input_ariaRequired could be different from the ariaRequired?